modules/cms/classes/Page.php

```diff
@@ -112,11 +115,25 @@ public static function getNameList()
         return $result;
     }
 
+    /**
+     * Helper that returns formatted robots tags.
+     * @return string
+     */
+    public function getMetaRobotsAttribute()
+    {
+        $options = [];
+        $options[] = $this->meta_robots_index ? 'noindex' : 'index';
+        $options[] = $this->meta_robots_follow ? 'nofollow' : 'follow';
+        $options[] = $this->meta_robots_archive ? 'noarchive' : 'archive';
```

Comment:
These properties should be prefixed with `no`: `$this->meta_robots_noindex` to reflect how they're actually used.